lib/index.js

```diff
@@ -118,25 +88,11 @@ function getCacheKey(context) {
  */
 function makeYCB(config, dimensions, contents) {
     var ycbBundle,
-        ycb,
-        originalRead,
-        originalReadNoMerge;
-
+        ycb;
     contents = contents || {};
     ycbBundle = [{dimensions: dimensions}];
-    // need to copy contents, since YCB messes with it
-    ycbBundle = ycbBundle.concat(clone(contents));
+    ycbBundle = ycbBundle.concat(contents);
```

Comment:
it was cloning `contents` previously.  curious why is it not needed any more.

Reply:
All current major versions of YCB clone the input so it's no longer needed.
https://github.com/yahoo/ycb/commit/f1c52069e18e074ccf271ae7e9cfd2a5fbc024b5